js/src/views/Signer/store.js

```diff
@@ -95,7 +95,11 @@ export default class SignerStore {
     this._api.parity
       .localTransactions()
       .then((localTransactions) => {
-        this.setLocalHashes(Object.keys(localTransactions));
+        const keys = Object
+          .keys(localTransactions)
+          .filter((key) => localTransactions[key].status !== 'invalid');
```

Comment:
Huh. I don't know how else to remove canceled transactions. When removeTransaction is called, it does not actually remove the tx from localTransactions, but rather gives 'invalid', so editing and cancelling tx, they don't go away...

Like we discussed before. To avoid having to change the gasPrice, I completely cancel the TX and then resubmit. It works beautifully. If you hit cancel, the submittal bar pops up with every single detail from the just cancelled tx still there. So you just can sign and resubmit for the exact tx back again.